Header header
ObjectPose[] object_poses